charts/karmada-operator/values.yaml

```diff
@@ -80,3 +80,34 @@ operator:
     rollingUpdate:
       maxUnavailable: 0
       maxSurge: 50%
+
+## operator service config
+service:
+  ## @param service.type
+  type: ClusterIP
+  ## @param service.port port of metrics a port
+  port: 8080
```

Comment:
Shall we give it a meaningful port name?

Reply:
Got it